app/view/account_type.py

```diff
@@ -0,0 +1,107 @@
+from app import db
+from app.Schema.bank_account_schema import account_type_schema
+from app.model.bank_account import AccountType
+from flask import request
+from flask_restful import Resource
+from app.common.ResponseGenerator import ResponseGenerator
+from flask_api import status
+from app.common.logging import *
+
+
+class AccountTypeDetails(Resource):
+
+    def post(self):
+
+        """Create account_type in the AccountType table"""
+
+        account_data = request.get_json()
+        result = account_type_schema.validate(account_data)
+        if result:
+            logger.exception("Missing or sending incorrect data to create an activity")
```

Comment:
Please add `result` in the exception message and in the response in all the places.